### PR TITLE
Added CAN bus support in python lib

### PR DESCRIPTION
**Pull request by rahulbhivare**
Standalone CAN bus program is tested on QA7 board.
The code functionalities:
      1. CAN message receive thread is running on user defined can port. This thread pushes received data in a buffer.
      2.  User can call "api_pa_pc_read_can_data" function to read single data.
      3. To send data, user can use "api_pa_pc_send_can_message" function

### CODE REVIEW

File: examples/app-python/example_app.py [hunk 1]
```diff
@@ -156,6 +215,7 @@ def new():
     app.mount_sequence("UARTLoopback", ctl.handle_uart_loopback)
     app.mount_sequence("StageFile",ctl.handle_stage_filedownload)
     app.mount_sequence("PowerControl", ctl.handle_power_control)
+    app.mount_sequence("TestCanBus", ctl.handle_test_can_bus)
```
Comment (bcwaldon) [inline examples/app-python/example_app.py:218]: Minor suggestion - capitalize CAN here so it is clear

File: images/app-python/Dockerfile [hunk 2]
```diff
@@ -28,6 +28,7 @@ RUN python3 -m pip install --upgrade pip
 # This is required for GPIO and UART access using FTDI
 RUN apt install -y libftdi1
 RUN python3 -m pip install pyserial
+RUN python3 -m pip install python-can
```
Comment (bcwaldon) [inline images/app-python/Dockerfile:31]: Add a comment above this like the comment regarding GPIO/UART so it is clear this is for CAN bus interaction

File: lib/python/satos_payload_sdk/antaris_api_can.py [hunk 3]
```diff
@@ -0,0 +1,152 @@
+#
+#   Copyright 2022 Antaris, Inc.
```
Comment (bcwaldon) [inline lib/python/satos_payload_sdk/antaris_api_can.py:2]: Update to 2024, please

File: lib/python/satos_payload_sdk/antaris_api_can.py [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+        return g_GPIO_ERROR
+    
+    finally:
+        # Clean up and close the bus
+        bus.shutdown()
+
+def api_pa_pc_get_can_message_received_count():
+    return len(data_array)
+
+if __name__ == "__main__":
```
Comment (bcwaldon) [inline lib/python/satos_payload_sdk/antaris_api_can.py:115]: I don't think this test function belongs here. Please move it to an actual test/example file.

Comment (bcwaldon) [reply lib/python/satos_payload_sdk/antaris_api_can.py:115]: Or if the example_app.py already encompasses it, then you can just delete this

**Review by bcwaldon — COMMENTED** [review]
Hi @rahulbhivare, just some minor feedback here. I have not tested this myself so I trust everything is working on your end.

File: lib/python/satos_payload_sdk/antaris_api_can.py (hunk 4 above)
Comment (rahulbhivare) [reply lib/python/satos_payload_sdk/antaris_api_can.py:115]: I had added this function for standalone testing. But now I removed

File: lib/python/satos_payload_sdk/antaris_api_can.py (hunk 3 above)
Comment (rahulbhivare) [reply lib/python/satos_payload_sdk/antaris_api_can.py:2]: Done

File: images/app-python/Dockerfile (hunk 2 above)
Comment (rahulbhivare) [reply images/app-python/Dockerfile:31]: Yes. Added comment.

File: examples/app-python/example_app.py (hunk 1 above)
Comment (rahulbhivare) [reply examples/app-python/example_app.py:218]: Updaed sequence name

**Pull request merged**